### ISSUE TITLE
Doesn't map properly to DualShock 4 buttons

### ISSUE THREAD

**Issue by b3328** [Ryochan7/DS4Windows#625]
I am having trouble identifying the problem, but as far as I can tell DS4Windows is mapping the buttons incorrectly. For example, the Z-Axis is mapped to L2 and R2 on the controller. The D-Pad maps to the "Point of View Hat". The right stick appears to map correctly to the X and Y rotation, however the left stick maps to the box next to that information. I have no idea as to what the correct mapping would be, nor do I know how to change the mapping if that is the problem. No, the controller does not work properly with applications.

**Comment by b3328** [comment]
![360 Controller Properties](https://user-images.githubusercontent.com/48837486/54840566-aef23580-4ca3-11e9-8b2a-12a7ab0ef452.PNG)

**Comment by Ryochan7** [comment]
That is the correct mapping. What you are experiencing, especially with LT and RT being mapped to one axis, are the results of design decisions made by Microsoft for the DirectInput mapping of the Xbox 360 controller. The big problem with the mapping is mapping LT and RT to one axis rather than mapping the triggers as two axes. That can create problems for some applications but past issues here have shown that some games using DirectInput would have problems if the triggers were mapped as two axes. Games need to implement XInput support to fully utilize the Xbox 360 controller.

In summary, this is not a problem with DS4Windows but an issue created by Microsoft.

**Comment by b3328** [comment]
Alright. Thanks for the help.
